### ISSUE TITLE
Difference between Nord and Northem 

### ISSUE THREAD

**Issue by adiospace** [arcticicestudio/nord#12]
Hi guys,

Thank you for the atom themes. They are awesome. 

Can you tell me which is the difference between Nord theme and Northem? Is Northem the precursor of Nord? Or is it a separate theme?

**Comment by arcticicestudio** [comment]
Hi Adrian,
nice to hear you like it.

Northem can be seen as a kind of precursor, but basically they are separate themes.
Nord has been designed with more precisely calculated colors and a focus on the bluish ambient. It is also generally more advanced and provides a lot of port projects. Nord will be updated frequently and is now the main color palette project.

There will be hotfixes for Northem port projects, but they will not be continued. Maybe some Northem port projects will be created by adopt it from the specific Nord project, but this is not clear yet.

In summary you should prefer Nord since it is the "better" and more advanced color palette.
